ui/src/components/drafts/DraftsItemIndex.js

```diff
@@ -71,38 +56,27 @@ class DraftsItemIndex extends React.Component {
       );
 
     let { draft_id } = this.props.match.params;
-    let isDraft = this.props.status == "draft" ? true : false;
-    let isPublishedOnce = this.props.recid ? true : false;
 
     return (
       <Box flex={true} direction="row" wrap={false}>
         <Box flex={false} colorIndex="grey-4">
-        <ReactTooltip />
-          <Anchor icon={<DocumentIcon/>} path={`/drafts/${draft_id}/edit`} data-tip="Edit metadata" />
+          <ReactTooltip />
+          <Anchor
+            icon={<DocumentIcon />}
+            path={`/drafts/${draft_id}/edit`}
+            data-tip="Edit metadata"
+          />
           <Anchor
-            icon={<ConnectIcon/>}
+            icon={<ConnectIcon />}
             path={`/drafts/${draft_id}/integrations`}
             data-tip="Connect your repositories"
           />
           {/*<Anchor icon={<PlayIcon />} path={`/drafts/${draft_id}/workflows`} data-tip="Run workflows" />*/}
           <Anchor
-            icon={<ShareIcon/>}
+            icon={<SettingIcon />}
             path={`/drafts/${draft_id}/settings`}
-            data-tip="Share with others"
+            data-tip="Analysis Settings"
```

Comment:
all of the actions in the sidebar are verbs, so we have:
edit metadata
connect your repositories
analysis settings (doesn't really fit here)